.github/workflows/prettier.yml

```diff
@@ -0,0 +1,16 @@
+name: Prettier
+
+on: push
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Prettify code
+        uses: creyD/prettier_action@v4.3
+        with:
+          prettier_plugins: "prettier-plugin-svelte"
+          prettier_options: --config .prettierrc --check .
```

Comment:
I think it's missing the `dry: true` flag

Reply:
As previously discussed, I am not entirely sure why we need to use the `dry=true` option if we're already using the `--check` config, but I've added it nonetheless.